src/mito2/src/sst/parquet.rs

```diff
@@ -399,4 +408,77 @@ mod tests {
         let mut reader = builder.build().await.unwrap();
         check_reader_result(&mut reader, &[new_batch_by_range(&["b", "h"], 150, 200)]).await;
     }
+
+    fn customize_column_config(
+        builder: WriterPropertiesBuilder,
+        region_metadata: &RegionMetadataRef,
+    ) -> WriterPropertiesBuilder {
+        let ts_col = ColumnPath::new(vec![region_metadata
+            .time_index_column()
+            .column_schema
+            .name
+            .clone()]);
+        let seq_col = ColumnPath::new(vec![SEQUENCE_COLUMN_NAME.to_string()]);
+
+        builder
+            .set_column_encoding(seq_col.clone(), Encoding::DELTA_BINARY_PACKED)
+            .set_column_dictionary_enabled(seq_col, false)
+            .set_column_encoding(ts_col.clone(), Encoding::DELTA_BINARY_PACKED)
+            .set_column_dictionary_enabled(ts_col, false)
+    }
+
+    #[tokio::test]
+    async fn test_read_large_binary() {
+        let mut env = TestEnv::new();
+        let object_store = env.init_object_store_manager();
+        let handle = sst_file_handle(0, 1000);
+        let file_path = handle.file_path(FILE_DIR);
+
+        let write_opts = WriteOptions {
+            row_group_size: 50,
+            ..Default::default()
+        };
+
+        let metadata = build_test_binary_test_region_metadata();
+        let json = metadata.to_json().unwrap();
+        let key_value_meta = KeyValue::new(PARQUET_METADATA_KEY.to_string(), json);
+
+        let props_builder = WriterProperties::builder()
+            .set_key_value_metadata(Some(vec![key_value_meta]))
+            .set_compression(Compression::ZSTD(ZstdLevel::default()))
+            .set_encoding(Encoding::PLAIN)
+            .set_max_row_group_size(write_opts.row_group_size);
+
+        let props_builder = customize_column_config(props_builder, &metadata);
+        let writer_props = props_builder.build();
+
+        let write_format = WriteFormat::new(metadata.clone());
+        let string = file_path.clone();
+        let mut buffered_writer = BufferedWriter::try_new(
+            string,
+            object_store.clone(),
+            write_format.arrow_schema(),
```

Comment:
The arrow schema created by the `WriteFormat` will use `Binary`type, let's map it to the `LargeBinary` type.
```suggestion
        let write_format = WriteFormat::new(metadata.clone());
        let fields: Vec<_> = write_format.arrow_schema().fields().map(|field| if field.name() == Field::new(field.name(), DataType::LargeBinary, field.is_nullable()) else { field.clone() }).collect();
        let arrow_schema = Arc::new(Schema::from(fields));
        let string = file_path.clone();
        let mut buffered_writer = BufferedWriter::try_new(
            string,
            object_store.clone(),
            arrow_schema,
```